### PR TITLE
Ug content for Syncing form fields

### PR DESCRIPTION
**Pull request by KrithikaGanesan**
Ug content for Syncing form fields

### CODE REVIEW

File: wpf/Pdf-Viewer/Form-Filling-in-PDF.md [hunk 1]
```diff
@@ -145,6 +145,76 @@ End Sub
 {% endhighlight %}
 {% endtabs %}
 
+## Add form fields at runtime
+
+The PDF Viewer allows users to programmatically add form fields without user interaction at runtime. After adding the form field, user can programmatically change the form field’s properties during runtime.
```
Comment (Vikassekar) [inline wpf/Pdf-Viewer/Form-Filling-in-PDF.md:150]: under add category no need to mention properties related things.

Comment (Vikassekar) [reply wpf/Pdf-Viewer/Form-Filling-in-PDF.md:150]: add separate para for property changes.

**Review by Vikassekar — CHANGES_REQUESTED** [review]

File: wpf/Pdf-Viewer/Form-Filling-in-PDF.md (hunk 1 above)
Comment (KrithikaGanesan) [reply wpf/Pdf-Viewer/Form-Filling-in-PDF.md]: I have updated as mentioned

**Review by Vikassekar — APPROVED** [review]
approving this.

**Pull request merged**